astroid/brain/brain_functools.py

```diff
@@ -101,7 +105,7 @@ def _functools_partial_inference(node, context=None):
     partial_function = objects.PartialFunction(
         call,
         name=inferred_wrapped_function.name,
-        doc=inferred_wrapped_function.doc,
+        doc=getattr(inferred_wrapped_function.doc_node, "value", None),
```

Comment:
```suggestion
```

Reply:
This depends on order of deprecation. See other comments in other PRs.